 static void do_pad(unsigned char value, char up_flag)
diff -urNp --exclude-from=/home/mitica/quintela/config/misc/dontdiff --exclude='defconfig*' --exclude='log.*' --exclude='log2?' w2/drivers/char/sbc60xxwdt.c w1/drivers/char/sbc60xxwdt.c
--- w2/drivers/char/sbc60xxwdt.c	2002-11-29 00:53:12.000000000 +0100
+++ w1/drivers/char/sbc60xxwdt.c	2003-08-19 01:36:50.000000000 +0200
@@ -335,7 +335,7 @@ err_out_region2:
 	release_region(WDT_START,1);
 err_out_region1:
 	release_region(WDT_STOP,1);
-err_out:
+//err_out:
 	return rc;
 }
 
